apps/emqx_auth_kerberos/test/emqx_authn_kerberos_SUITE.erl

```diff
@@ -69,6 +69,26 @@ t_create(_Config) ->
     {ok, [#{provider := emqx_authn_kerberos}]} =
         emqx_authn_chains:list_authenticators(?GLOBAL).
 
+t_disable_without_kinit(_Config) ->
```

Comment:
This seems unrelated with changes herein?

Reply:
this is because i changed merge base from release-58 to master.
they should be merged anyway.